packages/checkout/widgets-lib/src/lib/walletConnect.ts

```diff
@@ -46,18 +50,20 @@ const lightThemeVariables = {
 const metamaskId = 'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96';
 const frontierId = '85db431492aa2e8672e93f4ea7acf10c88b97b867b0d373107af63dc4880f041';
 const ledgerLiveId = '19177a98252e07ddfc9af2083ba8e07ef627cb6103467ffebb3f8f4205fd7927';
-// const coinbaseId = 'fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa';
+const coinbaseId = 'fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa';
 // const phantomId = 'a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393';
 // const rainbowId = '1ae92b26df02f0abca6304df07debccd18262fdf5fe82daa81593582dac9a369';
 
 const productionWalletWhitelist = [metamaskId, frontierId, ledgerLiveId];
-const sandboxWalletWhitelist = [metamaskId];
+const sandboxWalletWhitelist = [metamaskId, frontierId, coinbaseId];
```

Comment:
Not sure what wallets we want in this array

Reply:
frontier doesn't work in Sandbox as it doesn't have Sepolia so it shouldn't be there. I haven't tested with coinbase mobile wallet either. Can we check this?

Reply:
Good catch @sharonsheah .. I've removed these both from sandbox!